plugins/modules/aws_glue_job.py

```diff
@@ -47,6 +53,11 @@
     description:
       - Description of the job being defined.
     type: str
+  glue_version:
+    description:
+      - Glue version determines the versions of Apache Spark and Python that AWS Glue supports.
+    type: str
+    version_added: 1.5.0
```

Comment:
There is one old version_added.

Reply:
existed already, just got moved in the docs.